.github/CONTRIBUTING.md

```diff
@@ -10,7 +10,11 @@ maintain it and you contribute.
 
 2. pylint (lints all the things)
 
-3. black (opinionated formatter)
+.. note:: In early development cycles, a decision was made to use black as a formatter
+          which is why current pull-requests are required to pass a
+         `black --diff` check of the source tree. The decision to use `black` is
+         left to individual developers as the formatting changes it makes can be
+         achieved without it.
```

Comment:
There are two problems with this. The first one is that typically in RST/Sphinx the prose goes below the directive which allows to make better use of the space:
```suggestion
.. note::

   In early development cycles, a decision was made to use black as a formatter
   which is why current pull-requests are required to pass a
   `black --diff` check of the source tree. The decision to use `black` is
   left to individual developers as the formatting changes it makes can be
   achieved without it.
```
The second problem is that you're adding RST inside Markdown which won't work. You could use MyST syntax to make it work within Sphinx, at least, though:
````suggestion
```{note}
In early development cycles, a decision was made to use black as a formatter
which is why current pull-requests are required to pass a
`black --diff` check of the source tree. The decision to use `black` is
left to individual developers as the formatting changes it makes can be
achieved without it.
```
````

Reply:
awesome, thanks! I totally forgot about MyST part of it